Sources/Roadmap/RoadmapFeatureView.swift

```diff
@@ -33,7 +33,8 @@ struct RoadmapFeatureView: View {
                 Text(viewModel.feature.localizedFeatureTitle)
                     .font(viewModel.configuration.style.titleFont)
                 
-                if let description = viewModel.feature.localizedFeatureDescription {
+                let description = viewModel.feature.localizedFeatureDescription
+                if description != "" {
```

Comment:
```suggestion
                if !description.isEmpty {
```
It works, but this gives consistency with line 85/87